src/zadatak1/Vlasnik.java

```diff
@@ -0,0 +1,44 @@
+package zadatak1;
+
+public class Vlasnik {
+
+    private String imePrezime;
+    private String jmbg;
+    private String brojLicneKarte;
+
+    public Vlasnik(String imePrezime, String jmbg, String brojLicneKarte) {
+
+            if (!imePrezime.isEmpty() && ! imePrezime.matches("(.*)[!@#$%^&*(){};:',.<>?/`~\\d](.*)")) {
```

Comment:
static final za regex

 imePrezime.matches("^[a-zA-Z'\\-\\s]+$")) izgleda mnogo lepse a plus moze i da se proda u Irskoj ;)  

https://stackoverflow.com/questions/44420498/java-android-regex-expression-to-allow-names-that-may-have-apostrophe-and-or-hyp